src/components/BuildConfigsList/BuildConfigsList.tsx

```diff
@@ -137,9 +137,7 @@ export const BuildConfigsList = ({
                     </Th>
                   )}
                 {columns.includes(buildConfigEntityAttributes.actions.id) && (
-                  <ProtectedComponent>
-                    <Th width={15}>{buildConfigEntityAttributes.actions.title}</Th>
-                  </ProtectedComponent>
+                  <Th width={15}>{buildConfigEntityAttributes.actions.title}</Th>
```

Comment:
React 18 has enhanced validation for incorrect HTML structures, ensuring Th and Td elements are only placed inside Tr, Thead, Tbody, and Table. ProtectedComponent in a table cell caused additional div elements and I think as a table header there is no need to wrap it with ProtectedComponent